BootloaderCorePkg/Tools/PrepareBuildComponentBin.py

```diff
@@ -120,7 +93,7 @@ def CopyFspBins (fsp_dir, sbl_dir, platform):
           ('CoffeeLakeFspBinPkg/SampleCode/Vbt/Vbt.bsf', 'Platform/CoffeelakeBoardPkg/VbtBin/Vbt.bsf')
         ])
     else:
-        Fatal ('%s platform is not supported yet !' % platform)
```

Comment:
Can you consider leaving this line here but just change it to 'print' instead of 'Fatal' ? I also have some build issue because of this one.

Reply:
It's okay to just remove it if preferred, I don't have a strong opinion on this one really, just want this fix.